bin/update_dev_data.py

```diff
@@ -143,6 +143,13 @@ def devdata():
                 os.path.join(pathlib.Path(lms.__file__).parent.parent, ".devdata.env"),
             )
 
+            # Copy any certfificates
+            shutil.copytree(
```

Comment:
This is sorter that copying over two specific files and more flexible for the future.

Reply:
:+1: Good idea. I don't think we actually made the change but I think Jon suggested on a previous PR a while ago that every devdata dir could have a files dir for files that get blindly copied over to their corresponding locations in the app. So this would work for any files not just certificates. Something like this in the devdata repo:

```
devdata/
  lms/
    devdata.json  # Doesn't get copied.
    files/
      certificates/
        cert.txt  # Gets copied to certificates/cert.txt in the app.
      foo/
        bar/
          gar.txt  # Gets copied to foo/bar/gar.txt in the app
```